pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -793,6 +871,38 @@ func (n *NetworkPolicyController) deleteNetworkPolicy(old interface{}) {
 	n.deleteDereferencedAddressGroups(oldInternalNP)
 }
 
+// addNode receives Node ADD events and reprocesses the ClusterNetworkPolicy
+// with `toService` referred to NodePort type Services.
+func (n *NetworkPolicyController) addNode(obj interface{}) {
+	defer n.heartbeat("addNode")
+	// Find all ACNPs that are impacted by this Node and reprocess them.
+	affectedACNPsByService := n.filterACNPsByServiceType(v1.ServiceTypeNodePort)
+	for _, acnp := range affectedACNPsByService {
+		n.reprocessACNP(acnp)
+	}
+}
+
+// updateNode receives Node UPDATE events and reprocesses the ClusterNetworkPolicy
+// with `toService` referred to NodePort type Services.
+func (n *NetworkPolicyController) updateNode(oldObj, curObj interface{}) {
```

Comment:
We don't care about a Node update event that's not an IP address change (or from `Ready` to `NotReady` maybe), otherwise we could be hit with a high volume of ACNP resync that is not needed

Reply:
Good point! Done.